config/locales/de.yml

```diff
@@ -994,23 +994,18 @@ de:
     difficulty_ok: "Die Aufgabenschwierigkeit war genau richtig."
     difficulty_some_what_difficult: "Die Aufgabe war etwas zu schwierig."
     difficult_too_difficult: "Die Aufgabe war viel zu schwierig."
-    difficulty: "Schwierigkeit der Aufgabe:"
```

Comment:
Should I remove the answer options as well?
And should I remove the lines (232 - 234)?
```
pair_programming_exercise_feedback:
        one: Feedback
        other: Feedbacks
```

Reply:
I would say so, yes (for both, the answer options and the model human name).